src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,31 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * {@code Deadline} is a type of {@code Task} where user must specify the description and the "/by" remark
+ */
+public class Deadline extends Task {
+
+    protected LocalDate date;
+
+    /**
+     * Constructor for {@code Deadline}
+     * @param description the description of the {@code Deadline}
+     * @param date the remark of the {@code Deadline}
+     */
+    public Deadline(String description, LocalDate date) {
+        super(description);
+        this.date = date;
```

Comment:
Perhaps this can be expanded to accept more variations of date inputs as well as to include time